src/native/webrtc.rs

```diff
@@ -0,0 +1,375 @@
+//                     self.uuid = req.uuid;
+//                     let sdp_decoded = self.engine.decode(s.sdp)?;
+//                     let sdp_decoded: SdpOffer = serde_json::from_slice(sdp_decoded.as_slice())?;
+//                     log::debug!(
+//                         "Getting offer {:?} of type {:?}",
+//                         &sdp_decoded.sdp,
+//                         &sdp_decoded.r#type
+//                     );
+//                     let mut cursor = Cursor::new(sdp_decoded.sdp);
+//                     log::debug!("we have an offer - there should be no PeerConnection");
+//                     if self.peer_connection.lock().unwrap().is_some() {
+//                         return Err(anyhow::anyhow!("a peer connection already exists"));
+//                     }
+//                     let sdp = sdp::SessionDescription::unmarshal(&mut cursor)?;
+//                     let _ = self
+//                         .peer_connection
+//                         .lock()
+//                         .unwrap()
+//                         .insert(PeerConnection::new(sdp));
+
+//                     let answer = if let Some(cp) = self.peer_connection.lock().unwrap().as_ref() {
+//                         cp.answer()
+//                     } else {
+//                         return Err(anyhow::anyhow!(
+//                             "No peerconnection found but we just created is"
+//                         ));
+//                     };
+
+//                     log::debug!("We are answering {:?}", &answer);
+
+//                     let answer = SdpOffer {
+//                         r#type: "answer".to_owned(),
+//                         sdp: answer.marshal(),
+//                     };
+
+//                     let answer = self.engine.encode(serde_json::to_string(&answer)?);
+
+//                     let answer = AnswerResponse {
+//                         uuid: self.uuid.clone(),
+//                         stage: Some(rpc::webrtc::v1::answer_response::Stage::Init(
+//                             AnswerResponseInitStage { sdp: answer },
+//                         )),
+//                     };
+//                     return Ok(Some(answer));
+//                 }
+//                 rpc::webrtc::v1::answer_request::Stage::Update(u) => {
+//                     if let Some(c) = u.candidate {
+//                         //log::info!("getting {:?}", &c.candidate);
+//                         let candidate: Candidate = c.candidate.try_into()?;
+//                         if let Some(c) = &self.remote_candidate_chan_tx {
+//                             //log::info!("sendind candidate");
+//                             c.send(candidate).await?;
+//                         }
+//                     }
+//                 }
+//                 _ => return Err(anyhow::anyhow!("not yet implemented")),
+//             }
+//         }
+//         Ok(None)
+//     }
+// }
```

Comment:
This entire file appears to be brand new and completely commented out. Is it supposed to be here?